tests/momento/simple_cache_client/test_dictionary_async.py

```diff
@@ -0,0 +1,963 @@
+            assert isinstance(remove_response, CacheDictionaryRemoveField.Success)
+
+            fetch_response = await client_async.dictionary_fetch(cache_name, dictionary_name)
+            assert isinstance(fetch_response, CacheDictionaryFetch.Miss)
+
+
+# TODO these don't work for this case?
+# @behaves_like(a_cache_name_validator)
+# @behaves_like(a_connection_validator)
+@behaves_like(a_dictionary_remover)
+@behaves_like(a_dictionary_name_validator)
+@behaves_like(a_dictionary_fields_validator)
+def describe_dictionary_remove_fields() -> None:
+    @fixture
+    def cache_name_validator(
+        client_async: SimpleCacheClientAsync,
+        dictionary_name: TDictionaryName,
+        dictionary_fields: TDictionaryFields,
+    ) -> TCacheNameValidator:
+        return partial(client_async.dictionary_remove_fields, dictionary_name=dictionary_name, fields=dictionary_fields)
+
+    @fixture
+    def connection_validator(
+        dictionary_name: TDictionaryName, dictionary_fields: TDictionaryFields
+    ) -> TConnectionValidator:
+        async def _connection_validator(client_async: SimpleCacheClientAsync, cache_name: TCacheName) -> CacheResponse:
+            return await client_async.dictionary_remove_fields(
+                cache_name, dictionary_name=dictionary_name, fields=dictionary_fields
+            )
+
+        return _connection_validator
+
+    @fixture
+    def dictionary_fields_validator(
+        client_async: SimpleCacheClientAsync,
+        cache_name: TCacheName,
+        dictionary_name: TDictionaryName,
+    ) -> TDictionaryNameValidator:
+        return partial(client_async.dictionary_remove_fields, cache_name=cache_name, dictionary_name=dictionary_name)
+
+    @fixture
+    def dictionary_name_validator(
+        client_async: SimpleCacheClientAsync,
+        cache_name: TCacheName,
+        dictionary_fields: TDictionaryFields,
+    ) -> TDictionaryNameValidator:
+        return partial(client_async.dictionary_remove_fields, cache_name=cache_name, fields=dictionary_fields)
+
+    @fixture
+    def dictionary_remover(cache_name: TCacheName) -> TDictionaryRemover:
+        async def _dictionary_remover(
+            client_async: SimpleCacheClientAsync,
+            dictionary_name: TDictionaryName,
+            field: TDictionaryField,
+        ) -> CacheResponse:
+            return await client_async.dictionary_remove_fields(cache_name, dictionary_name, [field])
+
+        return _dictionary_remover
+
+    async def it_removes_multiple_items(
+        client_async: SimpleCacheClientAsync,
+        cache_name: TCacheName,
+        dictionary_name: TDictionaryName,
+        dictionary_items: TDictionaryItems,
+    ) -> None:
+        set_response = await client_async.dictionary_set_fields(
+            cache_name, dictionary_name, dictionary_items, CollectionTtl()
+        )
+        assert isinstance(set_response, CacheDictionarySetFields.Success)
```

Comment:
Sprinkle in some other items to make sure it's not just deleting the whole dictionary.

Reply:
Sprinkled in [971c643](https://github.com/momentohq/client-sdk-python/pull/237/commits/971c64300d0dd526bfdedda5db75b05da172adf6)